### DISCUSSION TITLE
Authorization Token : Bearer or API key in Environment ?

### DISCUSSION THREAD

**Discussion by kalimbra** [usebruno/bruno#823] [Q&A] [answered]
I am new at Bruno and a beginner in the use of API, but I try to test our developers API using environment vars to authenticate, but it seems I can't pass the "value" of authorization from environment to the call. 

If I set the var in "Auth", it pass a Bearer (which in some case is not the case) but do not replace the var 
![2023-10-30_10-45](https://github.com/usebruno/bruno/assets/7127212/4f3c8ea2-702f-467b-b116-95202bc2ca1f)

![2023-10-30_10-46](https://github.com/usebruno/bruno/assets/7127212/e421d5c8-b7ff-4d6f-b56a-a213634efd9e)

So I tried to pass it as a Header but get the same issue : 
![2023-10-30_10-48](https://github.com/usebruno/bruno/assets/7127212/d9454d54-8a6e-4bde-931c-23ea3a6670ca)

The only way to make it work it to replace the value `{{Authorizaton}}` with the real value. 

Please keep in mind that I am a complete beginner, so I certainly do something wrong, be kind 🙏

**Comment by Kaargo1** [comment]
It's possible that this was a bug(although I can't find a fix in the releases). It works for me in the current version(1.2.0).
Does it work if you update Bruno to 1.2.0?

**Reply by kalimbra** [reply, marked as answer]
Apparently the issue is related to the VAR name directly. If I use {{Authorization}} it does not work, but {{AuthCode}} or {{ApiKey}} or anything else seems to work correctly. 

Perhaps the fact that there is "Authorization" : {{Authorization}} is the issue. 

Anyway, seems to work correctly this way, I wll resolve these issue.

**Comment by johanehr** [comment]
I encountered a similar issue in 1.20.0, where I wanted to set a property in a JSON body with an environment variable called the same thing, apparently returning undefined (when hovering over an error in the text box). Changing the variable name to something different seemed to make it work.

Example:
Body:
```
{
  "propertyName": "{{propertyName}}"
}
```

local.bru
```
vars:secret [
  propertyName
]
```

I was unable to replicate it now, but perhaps it can help in pin-pointing a potential bug!
